ap2_houston/ap2_houston/servicer/servicer.py

```diff
@@ -17,28 +17,39 @@ class Servicer(APIServiceServicer):
     def GetIncident(self, request, context):
         log.info("GetIncident")
         context.set_code(grpc.StatusCode.OK)
+        #returns a list of Incident objects
+        incidents = get_incidents()
+        #for each incident object, break it down
+        new_incidents = []
+        for incident in incidents: 
+            new_incident = Incident(
+                title=getattr(incident,'title'),
+                problem_summary=getattr(incident,'problem_summary'),
+                severity=getattr(incident,'severity'),
+                status=getattr(incident,'status'),
+                state=getattr(incident,'state'),
+                slack_channel_id=getattr(incident,'slack_channel_id'),
+                slack_channel_name=getattr(incident,'slack_channel_name'),
+                incident_commander=getattr(incident,'incident_commander'),
+                communications=getattr(incident,'communications'),
+                scribe=getattr(incident,'scribe'),
+                open_date=getattr(incident,'open_date'),
+                update_date=getattr(incident,'update_date'),
+                resolve_date=getattr(incident,'resolve_date'),
+                resolution_summary=getattr(incident,'resolution_summary'),
+                incident_aware_date=getattr(incident,'incident_aware_date'),
+                incident_aware_time=getattr(incident,'incident_aware_time'),
+                impact_start_date=getattr(incident,'impact_start_date'),
+                impact_start_time=getattr(incident,'impact_start_time'),
+                last_incident_update=getattr(incident,'last_incident_update'),
+                subject_matter_experts=getattr(incident,'subject_matter_experts'),
+                #timeline=getattr(incident,'timeline'),
+                #recipients=getattr(incident,'recipients'),
```

Comment:
```suggestion
                #timeline=getattr(incident,'timeline',[]),
                #recipients=getattr(incident,'recipients',{}),
```

You might be able to get around this by passing a default/fallback value